docs/01-Getting-started.md

```diff
@@ -35,7 +35,7 @@ $ composer require accompli/accompli:dev-master --dev
 
 ## Using Accompli
 
-In order for Accompli to run you require a accompli.json file in the root of your project. 
+In order for Accompli to run your project requires an accompli.json file in the root of your project. 
```

Comment:
Might be better to change 'your project' to 'the project' at the end of the sentence.